src/Libraries/DSCPython/CPythonEvaluator.cs

```diff
@@ -231,6 +235,39 @@ static CPythonEvaluator()
             }
         }
 
+        /// <summary>
+        /// Processes additional bindings that are not actual inputs.
+        /// Currently, only the node name is received in this way.
+        /// </summary>
+        /// <param name="scope">Python scope where execution will occur</param>
+        /// <param name="bindingNames">List of binding names received for evaluation</param>
+        /// <param name="bindingValues">List of binding values received for evaluation</param>
+        private static void ProcessAdditionalBindings(PyScope scope, IList bindingNames, IList bindingValues)
```

Comment:
The two function body here looks very similar, are they identical? Can we somehow move the shared part to PythonNodeModels project?

Reply:
I think the key part is the `somehow`. A month ago doing that would have been easy, but now the engines don't reference `PythonNodeModels` anymore. I guess we can revisit this as something to take into account when factoring out the engines and trying to re-use code between them. What do you think @QilongTang ?

Reply:
Ah sorry I keep forgetting about that we did so many changes to the projects.. Sure, let's consider this later